src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -258,13 +259,16 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                     ->end()
                                 ->end()
                                 ->validate()
-                                    ->always(function ($v) {
-                                        if (isset($v['type']) && isset($v['service'])) {
-                                            throw new \InvalidArgumentException('"type" and "service" could not be used together.');
-                                        }
-
-                                        return $v;
-                                    })
+                                    ->ifTrue(function ($v) { return !isset($v['type']) && !isset($v['service']); })
```

Comment:
I my previous PR, I added that too. But I realized it's not mandatory. If nothing is configured, we don't give a MarkingStore to the workflow, and so the workflow will use the default MarkingStore. So it makes the configuration simpler.